python/triton/runtime/autotuner.py

```diff
@@ -184,7 +185,8 @@ def prune_configs(self, kwargs):
                         num_warps=config.num_warps,
                         num_ctas=config.num_ctas,
                         enable_warp_specialization=config.enable_warp_specialization,
-                        enable_persistent=config.enable_persistent,
+                        # TODO: Make it configurable
+                        # enable_persistent=False,
```

Comment:
@jlebar following what did for `enable_persistent`

Reply:
Right

Reply:
sgtm